src/styles/helpers/normalizeThemeColors.js

```diff
@@ -0,0 +1,25 @@
+import { colorToRgba } from './color';
+
+export default function normalizeThemeColors(theme) {
+    const normalizePaletteColors = palette => {
+        const normalizedPalette = {};
+        Object.keys(palette).forEach(key => {
+            const value = palette[key];
+            if (typeof value === 'string') {
+                normalizedPalette[key] = colorToRgba(value);
+            } else if (typeof value === 'object') {
+                normalizedPalette[key] = normalizePaletteColors(palette[key]);
+            } else {
+                normalizedPalette[key] = palette[key];
+            }
+        });
+        return normalizedPalette;
+    };
```

Comment:
let's put this function outside normalizeThemeColors function but in this same file